test/utils/nonsipp/check/SharesCheckStatusUtilsSpec.scala

```diff
@@ -14,37 +14,22 @@
  * limitations under the License.
  */
 
-package utils.nonsipp
+package utils.nonsipp.check
 
 import pages.nonsipp.shares._
 import models.IdentityType._
-import pages.nonsipp.landorproperty._
-import eu.timepit.refined.refineMV
-import utils.nonsipp.CheckStatusUtils._
+import utils.nonsipp.check.SharesCheckStatusUtils.{checkSharesRecord, checkSharesSection}
+import org.scalatest.OptionValues
 import models._
 import pages.nonsipp.common._
 import models.IdentitySubject._
 import org.scalatest.matchers.must.Matchers
 import models.ConditionalYesNo._
-import config.RefinedTypes._
+import config.RefinedTypes.Max5000
 import controllers.ControllerBaseSpec
-import org.scalatest.OptionValues
-import uk.gov.hmrc.domain.Nino
-
-class CheckStatusUtilsSpec extends ControllerBaseSpec with Matchers with OptionValues {
```

Comment:
No changes were made to this Spec, I just copied the Shares related code from CheckStatusUtilsSpec and made this its own file.